<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyInteractiveSeedingModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyInteractiveSeedingModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>733</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tractography Interactive Seeding</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLNodeComboBox" name="ParameterNodeSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLTractographyInteractiveSeedingNode</string>
      </stringlist>
     </property>
     <property name="showHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parameter set</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InputsCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>IO</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input DTI Volume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="DTINodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLDiffusionTensorVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Input Fiducials, Model or Label Map</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="FiducialNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLAnnotationHierarchyNode</string>
          <string>vtkMRMLModelNode</string>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Output Fiber Bundle</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="FiberNodeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLFiberBundleNode</string>
         </stringlist>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="SeedOptionPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ctkCollapsibleButton" name="PlacementOptionCollapsibleButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Seed Placement Options</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="collapsed">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_1">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Fiducial Region Size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ctkSliderWidget" name="FiducialRegionSpinBox">
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>2.500000000000000</double>
            </property>
            <property name="suffix">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Fiducial Seeding Step Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ctkSliderWidget" name="FiducialStepSpinBox">
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="minimum">
             <double>0.200000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
            <property name="suffix">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="SeedSelectedFiducialsLabel">
            <property name="text">
             <string>Seed Selected Fiducials</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="SeedSelectedCheckBox">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="MaxNumberOfSeedsLabel">
            <property name="text">
             <string>Max Number of Seeds</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="MaxNumberSeedsNumericInput">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="LabelMapOptionPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="ctkCollapsibleButton" name="LabelMapOptionCollapsibleButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Label Map Options</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="collapsed">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="UseIndexSpaceLabel">
            <property name="text">
             <string>Use index Space</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="UseIndexSpaceCheckBox">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="SeedSpacingLevel">
            <property name="text">
             <string>Seed Spacing</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ctkSliderWidget" name="SeedSpacingSlider">
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="minimum">
             <double>0.200000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
            <property name="suffix">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="RandomGridLabel">
            <property name="text">
             <string>Random Grid</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="RandomGridCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="LinearMeasureLabel">
            <property name="text">
             <string>Linear Measure Start Threshold</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="ctkSliderWidget" name="LinearMeasureStartSlider">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="ROILabelLabel">
            <property name="text">
             <string>ROI Label</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="ROILabelInput">
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="WriteFibersLabel">
            <property name="text">
             <string>Write Fibers To Disk</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="WriteFibersCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="OutputDirectoryLabel">
            <property name="text">
             <string>Output Directory</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="ctkDirectoryButton" name="OutputDirectoryButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="FilePrefixLabel">
            <property name="text">
             <string>File Prefix</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="FilePrefixLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="SeedOptionCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Tractography Seeding Parameters</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Minimum Path Length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLSliderWidget" name="MinimumPathSpinBox">
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="quantity" stdset="0">
         <string>length</string>
        </property>
        <property name="unitAwareProperties">
         <set>qMRMLSliderWidget::Precision|qMRMLSliderWidget::Prefix|qMRMLSliderWidget::Suffix</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="MaximumPathSpinBoxLabel">
        <property name="text">
         <string>Maximum Path Length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLSliderWidget" name="MaximumPathSpinBox">
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>800.000000000000000</double>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="quantity" stdset="0">
         <string>length</string>
        </property>
        <property name="unitAwareProperties">
         <set>qMRMLSliderWidget::Precision|qMRMLSliderWidget::Prefix|qMRMLSliderWidget::Suffix</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Stopping Criteria</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="StoppingCriteriaComboBox">
        <item>
         <property name="text">
          <string>Fractional Anisotropy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear Measure</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stopping Value</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkSliderWidget" name="StoppingValueSpinBox">
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Stopping Track Curvature</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkSliderWidget" name="StoppingCurvatureSpinBox">
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>0.700000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Integration Step Length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="qMRMLSliderWidget" name="IntegrationStepSpinBox">
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
        <property name="suffix">
         <string>mm</string>
        </property>
        <property name="quantity" stdset="0">
         <string>length</string>
        </property>
        <property name="unitAwareProperties">
         <set>qMRMLSliderWidget::Precision|qMRMLSliderWidget::Prefix|qMRMLSliderWidget::Suffix</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ControlOptionCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Enabling Options</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="CreateTractsInitiallyAsLabel">
        <property name="text">
         <string>Create Tracts Initially As</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="DisplayTracksComboBox">
        <item>
         <property name="text">
          <string>Lines</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tubes</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="EnableSeedingCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="EnableSeedingTractsLabel">
        <property name="text">
         <string>Enable Seeding Tracts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ParameterNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>DTINodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>FiducialNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>FiberNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MinimumPathSpinBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MaximumPathSpinBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTractographyInteractiveSeedingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>IntegrationStepSpinBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
